src/plugins/filters.ts

```diff
@@ -77,6 +77,20 @@ export const Filters = {
     return date.format(format)
   },
 
+  /**
+   * Formats a date from unixtime into a human readable
+   * datetime with optional formats for today and
+   * future datetimes
+   */
+  formatAbsoluteDateTime: (datetime: number, todayFormat?: string, futureFormat?: string) => {
+    const date = _Vue.$dayjs(datetime * 1000)
+    // Including a year doesn't make sense as that'll be clear from context (even on newyears-related edge cases)
+    const defaultFormat = 'MMM D, h:mm A'
+    const appropriateSpecifiedFormat = date.isToday() ? todayFormat : futureFormat
+
+    return date.format(appropriateSpecifiedFormat || defaultFormat)
+  },
```

Comment:
There is a similar method above (`formatDateTime`), but I can justify the creation of this new method for a few reasons:

- It's never meant to give a humanized relative offset (eg: "a few minutes from now") so it doesn't cover a large use case of the other method
- The other method could be updated, but it would get quite complex with the new format arguments and logic changes while maintaining some form of backward compat

If there is disagreement there I can see about taking a pass at refactoring the other method

Reply:
This is all good.